### PR TITLE
schedulers: add hot schedule log

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

schedulers: add hot schedule log

### Check List

Tests

- Unit test

Related changes
- Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by nolouch** [comment]
Can you show a log example?

File: server/schedulers/hot_region.go [hunk 1]
```diff
@@ -1073,6 +1081,15 @@ func (bs *balanceSolver) buildOperators() ([]*operator.Operator, []Influence) {
...
 			operator.OpHotRegion)
+		srcLd := bs.stLoadDetail[bs.cur.srcStoreID].LoadPred.min()
+		dstLd := bs.stLoadDetail[bs.cur.dstStoreID].LoadPred.max()
+		peer := bs.cur.srcPeerStat
+		additionalInfos["srcByteRate"] = strconv.FormatFloat(srcLd.ByteRate, 'f', 2, 64)
+		additionalInfos["srcKeyRate"] = strconv.FormatFloat(srcLd.KeyRate, 'f', 2, 64)
+		additionalInfos["dstByteRate"] = strconv.FormatFloat(dstLd.ByteRate, 'f', 2, 64)
+		additionalInfos["dstKeyRate"] = strconv.FormatFloat(dstLd.KeyRate, 'f', 2, 64)
+		additionalInfos["peerByteRate"] = strconv.FormatFloat(peer.ByteRate, 'f', 2, 64)
+		additionalInfos["peerKeyRate"] = strconv.FormatFloat(peer.KeyRate, 'f', 2, 64)
```
Comment (disksing) [inline server/schedulers/hot_region.go:1092]: Looks it repeates twice. maybe it can move to L1109.

**Comment by lhy1024** [comment]
> Can you show a log example?
```
[2020/10/27 12:19:17.872 +08:00] [INFO] [hot_test.go:154] ["add operator"] [region-id=5] [operator="\"move-hot-write-peer {mv peer: store [3] to [6]} (kind:leader,region,hot-region, region:5(0,0), createAt:2020-10-27 12:19:17.872758164 +0800 CST m=+0.090603581, startAt:0001-01-01 00:00:00 +0000 UTC, currentStep:0, steps:[add learner peer 262 on store 6, promote learner peer 262 on store 6 to voter, transfer leader from store 3 to store 4, remove peer on store 3])\""] ["additional info"="{\"dstByteRate\":\"3145728.00\",\"dstKeyRate\":\"31457.20\",\"peerByteRate\":\"524288.00\",\"peerKeyRate\":\"0.00\",\"srcByteRate\":\"6291456.00\",\"srcKeyRate\":\"62914.50\"}"]
```

**Comment by rleungx** [comment]
IMO, a better log can be recording the generated reason for this operator. From the example, it's still hard to know why it's generated.

**Comment by lhy1024** [comment]
> IMO, a better log can be recording the generated reason for this operator. From the example, it's still hard to know why it's generated.

What kind of details do you think is better? I think it may be hard to describe an operator from select store to select region.

File: server/schedulers/hot_region.go [hunk 2]
```diff
@@ -1099,6 +1100,20 @@ func (bs *balanceSolver) buildOperators() ([]*operator.Operator, []Influence) {
...
+	additionalInfos = make(map[string]string, 6)
+	srcLd := bs.stLoadDetail[bs.cur.srcStoreID].LoadPred.min()
+	dstLd := bs.stLoadDetail[bs.cur.dstStoreID].LoadPred.max()
+	peer := bs.cur.srcPeerStat
+	additionalInfos["srcByteRate"] = strconv.FormatFloat(srcLd.ByteRate, 'f', 2, 64)
+	additionalInfos["srcKeyRate"] = strconv.FormatFloat(srcLd.KeyRate, 'f', 2, 64)
+	additionalInfos["dstByteRate"] = strconv.FormatFloat(dstLd.ByteRate, 'f', 2, 64)
+	additionalInfos["dstKeyRate"] = strconv.FormatFloat(dstLd.KeyRate, 'f', 2, 64)
+	additionalInfos["peerByteRate"] = strconv.FormatFloat(peer.ByteRate, 'f', 2, 64)
+	additionalInfos["peerKeyRate"] = strconv.FormatFloat(peer.KeyRate, 'f', 2, 64)
```
Comment (nolouch) [inline server/schedulers/hot_region.go:1113]: can you add the exp information?

**Review by disksing — APPROVED** [review]

File: server/schedulers/hot_region.go (hunk 2 above)
Comment (HunDunDM) [reply server/schedulers/hot_region.go:1113]: @lhy1024 ping

**Pull request closed without merging**